lib/frontend/src/Obelisk/Frontend.hs

```diff
@@ -77,6 +89,23 @@ data Frontend route = Frontend
   , _frontend_body :: !(forall js t m. ObeliskWidget js t route m => RoutedT t route m ())
   }
 
+baseTag :: forall route js t m. ObeliskWidget js t route m => RoutedT t route m ()
+baseTag = elAttr "base" ("href" =: "/") blank --TODO: Figure out the base URL from the routes
+
+removeHTMLConfigs :: JSM ()
+removeHTMLConfigs = void $ runMaybeT $ do
+  doc <- MaybeT currentDocument
+  hd <- MaybeT $ getHead doc
+  es <- collToList =<< getElementsByTagName hd ("script" :: Text)
+  for_ es $ \e -> do
```

Comment:
Can they all be beneath some element we can identify by a fixed ID instead of enumerating all `script` tags?

Reply:
They're already directly beneath HEAD, and the content model for HEAD doesn't exactly have the flexibility people are used to in the BODY